CMakeLists.txt

```diff
@@ -94,6 +94,7 @@ endif()
 
 option(TBB_TEST "Enable testing" ON)
 option(TBB_EXAMPLES "Enable examples" OFF)
+option(TBB_STRIP_SYMBOLS "Enable examples" OFF)
```

Comment:
1. Copy-paste description
2. In my opinion, name of this variable doesn't reflect what it actually does. It should highlight that it strips debug info, and it should do it only on Linux.